examples/triggers/trigger.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: trigger
+spec:
+  interceptors:
+    - cel:
+        filter: "header.match('X-GitHub-Event', 'pull_request')"
+        overlays:
+        - key: extensions.truncated_sha
+          expression: "body.pull_request.head.sha.truncate(7)"
+  bindings:
+  - ref: pipeline-binding
+  template:
+    name: pipeline-template
```

Comment:
Just a suggestion 

I see `triggers` folder added as part of this PR inside `examples` 
So can we add example like [github](https://github.com/tektoncd/triggers/tree/master/examples/github) so with few yaml apply we should able to create it 

Right now with this examples user can get an idea how to use `triggerRef` but they need to apply `pipeline-binding`, `pipeline-template` from other folder

Reply:
Good idea. Done that.